docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md

```diff
@@ -0,0 +1,56 @@
+# Convert PyTorch* BERT-NER to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_pytorch_specific_Convert_Bert_ner}
+
+## Download and Convert the Model to ONNX*
+
+To download a pre-trained model or train the model yourself, refer
+to the [instruction](https://github.com/kamalkraj/BERT-NER/blob/dev/README.md) in the
+BERT-NER model repository. The model with config files is stored in the directory with name `out_base`.
+
+To convert the model to ONNX* format create and run the script with the following content in the root
```

Comment:
To convert the model to ONNX* format create and run the script with the following content in the root
directory of the model repository. If you download the pre-trained model, you will need
to download [`bert.py`](https://github.com/kamalkraj/BERT-NER/blob/dev/bert.py) to run the script.-->

To convert the model to the ONNX* format, create and run the script with the following content in the root
directory of the model repository. If you download the pre-trained model, you need
to download [`bert.py`](https://github.com/kamalkraj/BERT-NER/blob/dev/bert.py) to run the script.